### PR TITLE
Feature/adding ci

### PR DESCRIPTION
**Pull request by marcelomarra**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,8 @@
     "stack-pack-request": "2.3.1"
   },
   "devDependencies": {
-    "@gp-technical/stack-pack-standard": "^2.3.6"
+    "@gp-technical/stack-pack-standard": "^2.3.6",
+    "@gp-technical/eslint-config-stack": "^1.0.0",
+    "@gp-technical/stack-pack-app": "^2.0.2"
```
Comment (ArthurSeaton) [inline package.json]: stack-pack-gpapi shouldn't need stack-pack-app. The latter is for use in the app not the api.

Also the linting looks like it's not been added quite correctly. You shouldn't need the reference to standard-engine in package.json (line 20ish). See here for details: https://github.com/gp-technical/eslint-config-stack/blob/master/README.md.

Plus the way the linting is invoked is the old way as well. Again the details are in the Readme for eslint-config-stack as above.

I'd also try making stack-pack-util a peer dep (plus add it as a dev dep). That way we don't end up with multiple versions of the same package across the app. Only the app itself supplies the package with the other packages that need it pulling it in as a peer dependency.

Comment (marcelomarra) [reply package.json]: Thanks @ArthurSeaton for the comprehensive README.md .

I am applying the changes, please let me know if it's ok now.

Not fully understand about the stack-pack-util yet, can we discuss that later?

**Review by heltucosta — DISMISSED** [review]
LGTM 👍

**Comment by marcelomarra** [comment]
@ArthurSeaton the newest commit was created using the automation.

Let me know if everything is ok, please.

**Review by ArthurSeaton — COMMENTED** [review]
You need to remove the precommit entry from package.json. It's done via `.huskyrc.js` now.

**Review by ArthurSeaton — APPROVED** [review]
Seem ok now. Looks to behave ok locally for me.

**Pull request closed without merging**